src/java.security.jgss/share/native/libj2gss/NativeUtil.c

```diff
@@ -453,15 +453,15 @@ jint getJavaErrorCode(int cNonCallingErr) {
 
 
 /* Throws a Java Exception by name */
-void throwByName(JNIEnv *env, const char *name, const char *msg) {
+static void throwByName(JNIEnv *env, const char *name, const char *msg) {
```

Comment:
Why can't you move the few lines of `throwByName()` into `gssThrowOutOfMemoryError()` and totally remove `throwByName()`?

Reply:
Updated as suggested, thanks.